faust/assignor/partition_assignor.py

```diff
@@ -284,6 +287,25 @@ def _perform_assignment(
         )
         return res
 
+    def _global_table_standby_assignments(
+            self,
+            assignments: dict) -> dict:
+        # Ensures all members have access to all changelog partitions
+        # as standbys, if not already as actives
+        global_table_topics = []
+        for each_table in self._table_manager.data.values():
+            # Add changelog standbys only if global table
+            if each_table.is_global:
+                changelog_topic_name = each_table._changelog_topic_name()
+                partitions = set(range(0, each_table.partitions))
+                global_table_topics.append(changelog_topic_name)
+                for each_client in assignments:
```

Comment:
same here, `for client in assignments`
Did you ever program in Apple Script per chance? :D

Reply:
Nope. I don't really know where I picked up that habit. 
Will change the variable name.